doc/api/crypto.md

```diff
@@ -1784,6 +1784,12 @@ otherwise `err` will be `null`. By default, the successfully generated
 `derivedKey` will be passed to the callback as a [`Buffer`][]. An error will be
 thrown if any of the input arguments specify invalid values or types.
 
+The `digest` should either be a string value or null:
+- if it's null, `sha1` will be the default one
```

Comment:
String literals should be enclosed in single quotes: `'sha1'`. Also the two instances of null above should be styled as `null`.